components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py

```diff
@@ -14,9 +14,17 @@
 """GCP launcher for batch prediction jobs based on the AI Platform SDK."""
 
 from . import job_remote_runner
+from .utils import artifact_util
+
+ARTIFACT_PROPERTY_KEY_BIGQUERY_OUTPUT_TABLE = 'bigqueryOutputTable'
+ARTIFACT_PROPERTY_KEY_BIGQUERY_OUTPUT_DATASET = 'bigqueryOutputDataset'
+ARTIFACT_PROPERTY_KEY_GCS_OUTPUT_DIRECTORY = 'gcsOutputDirectory'
 
 
 def create_batch_prediction_job_with_client(job_client, parent, job_spec):
+    # todo remove
+    print('**')
+    print(job_spec)
```

Comment:
nit: Seems like testing artifacts, should be removed?

Reply:
done